### PR TITLE
use as much webapps shortcut as possible

### PR DESCRIPTION
**Pull request by edevosc2c**
Everywhere where it's possible, use "webapps" shortcut.

Some places might still have possible refactoring but it would need more thoughts into it.

### CODE REVIEW

File: templates/geonetwork/ogc-api-records/ogc-api-records-deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.georchestra.webapps.geonetwork.enabled .Values.georchestra.webapps.geonetwork.ogc_api_records.enabled -}}
+{{- if and .Values.georchestra.webapps.geonetwork.enabled $webapp.enabled -}}
```
Comment (jeanpommier) [inline templates/geonetwork/ogc-api-records/ogc-api-records-deployment.yaml:1]: you're using webapp before defining it, here, no ?

File: templates/ldap/openldap-svc.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
-{{- if .Values.georchestra.webapps.openldap.enabled -}}
+{{- $webapp := .Values.georchestra.webapps.openldap -}}
```
Comment (jeanpommier) [inline templates/ldap/openldap-svc.yaml:1]: is it worth it here ? It's not even used after this initial test

Remark applies for other files too

File: templates/mapstore/mapstore-deployment.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-{{- if .Values.georchestra.webapps.mapstore.enabled -}}
-{{- $webapp := .Values.georchestra.webapps.mapstore -}}
+{{- if $webapp.mapstore.enabled -}}
+{{- $webapp := $webapp.mapstore -}}
```
Comment (jeanpommier) [inline templates/mapstore/mapstore-deployment.yaml:2]: duh ?
Defined before use in line 1. and this line 2 looks like a search'n'replace gone wild

File: templates/mapstore/mapstore-pvc.yaml [hunk 4]
```diff
@@ -1,4 +1,5 @@
-{{- if .Values.georchestra.webapps.mapstore.enabled -}}
+{{- if $webapp.mapstore.enabled -}}
```
Comment (jeanpommier) [inline templates/mapstore/mapstore-pvc.yaml:1]: same here, webapp not defined before

File: templates/mapstore/mapstore-svc.yaml [hunk 5]
```diff
@@ -1,4 +1,5 @@
-{{- if .Values.georchestra.webapps.mapstore.enabled -}}
+{{- if $webapp.mapstore.enabled -}}
```
Comment (jeanpommier) [inline templates/mapstore/mapstore-svc.yaml:1]: same

**Review by jeanpommier — CHANGES_REQUESTED** [review]
did you try deploying this PR ? I believe it shouldn't validate. There are several occurrences where the variable is defined before being used

File: templates/ldap/openldap-svc.yaml (hunk 2 above)
Comment (edevosc2c) [reply templates/ldap/openldap-svc.yaml:1]: well it's for future-proof. for the future in case we need to use the variable it's there, it's available.

**Comment by edevosc2c** [comment]
Oops I think I forgot to double-check, some webapps shortcut were not assigned correctly.

File: templates/mapstore/mapstore-deployment.yaml (hunk 3 above)
Comment (edevosc2c) [reply templates/mapstore/mapstore-deployment.yaml]: resolved

File: templates/geonetwork/ogc-api-records/ogc-api-records-deployment.yaml (hunk 1 above)
Comment (edevosc2c) [reply templates/geonetwork/ogc-api-records/ogc-api-records-deployment.yaml]: resolved

**Comment by jeanpommier** [comment]
Looks good to me. Did you deploy it for further checks ?

**Review by jeanpommier — APPROVED** [review]

**Comment by edevosc2c** [comment]
> Did you deploy it for further checks ?

Not yet I'll try. At least the github actions checks passed :).

**Comment by jeanpommier** [comment]
Yes. and helm doesn't complain. :o)

**Review by jeanmi151 — APPROVED** [review]
looks good 
but can you try/deploy it somewhere to validate it is correctly working before merging

**Comment by edevosc2c** [comment]
Validated by deploying georchestra helm chart in test cluster.

**Pull request merged**